ui/apps/platform/src/Components/Pagination/PaginationInput/PaginationInput.js

```diff
@@ -10,8 +10,8 @@ const TYPING_DELAY = 800;
 const PaginationInput = ({ totalSize, onChange, currentPage, pageSize }) => {
     const [localPage, setLocalPage] = useState(currentPage);
     const delayedSetPage = useCallback(
-        debounce((newPage) => onChange(newPage), TYPING_DELAY),
-        []
+        () => debounce((newPage) => onChange(newPage), TYPING_DELAY),
```

Comment:
For my understanding, was original `debounce(…)` incorrect? It does look like a subtle problem.

What are pro and con when arrow function does a side-effect instead of returns a value?
* implicitly `undefined` return value: `(newPage) => onChange(newPage), TYPING_DELAY)`
* explicitly no meaningful return value:

```js
(newPage) => {
    onChange(newPage), TYPING_DELAY);
}
```